docs/podman-create.1.md

```diff
@@ -610,6 +610,12 @@ It will also set the default stop signal to SIGRTMIN+3.
 
 This allow systemd to run in a confined container without any modifications.
 
+Note: On `SELinux` systems, systemd attempts to write to the cgroup
+file system.  Containers writing to the cgroup file system are denied by default.
+The `container_manage_cgroup` boolean must be enabled for this to be allowed on SELinux separated system.
```

Comment:
nit of 'a' nit:  s/on SELinux separated/on  a SELinux separated/

Reply:
FIxed to `allowed on an SELinux...`